### PR TITLE
add getting started with manager post

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

File: _posts/2016-05-26-Getting-started-with-the-new-manager.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+service. Once you are logged-in to the Alpha [Login](https://login.alpha.linode.com/login)
+server, you will need to create a new app. On the right-hand side of the screen
+is a box. Click Manage API clients. Then fill out the application name
+(Linode Manager) and callback URL (http://localhost:3000/oauth/callback). Then
+hit Register. Make sure to save the client id and secret to a file.
+
+Now clone the Manager [repo](https://github.com/linode/manager). Make sure npm is
+installed then run the following to pull in all dependencies:
+
+{% highlight bash %}
```
Comment (RobertDeRose) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: This is Markdown, not ReStructedText, you use backticks for code highlighting

    ```bash 
    code here
    ```

Comment (SirCmpwn) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: Both work.

Comment (eatonphil) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: @RobertDeRose tbh i don't know the first thing about Ruby template stuff. I just took it from existing [posts](https://raw.githubusercontent.com/linode/blog/gh-pages/_posts/2016-04-12-Announcing-APIv4.md).

Comment (RobertDeRose) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: Well backticks have the benefit of being properly rendered on Github when you click the "view" button.

File: _posts/2016-05-26-Getting-started-with-the-new-manager.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+As you may have [seen](https://engineering.linode.com/2016/05/16/Announcing-the-new-open-source-manager.html),
+the new Linode Manager is an entirely static app being developed in the
+[open](https://github.com/linode/manager) on Github. This post will walk you
+through the steps required to get the new Manager up and running on your local
+machine. This process can also be extended to building any application on the
+new API.
+
+Note: the Manager is in its infancy. At any point code during this time, pages
```
Comment (displague) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: `At any point during this time` (strike `code`).. `At any point` and `during this time` seem redundant.

Comment (eatonphil) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md]: Whoops, I thought I fixed that. Good catch.

**Pull request merged**